### ISSUE TITLE
[feature request] Deleted shares behaviour (receiver of a share)

### ISSUE THREAD

**Issue by laryllian** [owncloud/apps#1169]
Hello there,
when deleting a folder/file in the Shared folder (=the user is the receiver), it is simply gone, like a normal file/folder. This is not optimal, as I think.

In my view, it would be better to put shared files/folders in some kind of "inactive" mode when "deleted". They should then not behave like normal folders/files (and therefore not visible/synced by clients and such) anymore, but still be "there" and - this is important - could later be "activated" again.

To summarize, I would suggest that shares cannot be deleted, only deacivated. This would help me a lot and also earn tons of "thoughtful design"-points. ;-)

Any comments?
Laryllian

**Comment by icewind1991** [comment]
If this request is still valid please reopen in the core repo
